### ISSUE TITLE
HowTo example uses wrong types for "tool" and "supply" values

### ISSUE THREAD

**Issue by Aaranged** [schemaorg/schemaorg#1884]
The (mircodata) example on the [`HowTo` page](http://schema.org/HowTo) uses the wrong types for the value of [`tool`](http://schema.org/tool) and [`supply`](http://schema.org/supply).

```
<div itemprop="tool" itemscope itemtype="http://schema.org/HowToItem">Spare tire</div>
<div itemprop="tool" itemscope itemtype="http://schema.org/HowToItem">Lug wrench</div>
<div itemprop="tool" itemscope itemtype="http://schema.org/HowToItem">Jack</div>
<div itemprop="tool" itemscope itemtype="http://schema.org/HowToItem">Wheel wedges</div>
<div itemprop="supply" itemscope itemtype="http://schema.org/HowToItem">Flares</div>
```

The `itemtype` for `tool` should be [`HowToTool`](http://schema.org/HowToTool); the `itemtype` for `supply `should be [`HowToSupply`](http://schema.org/HowToSupply)

```
<div itemprop="tool" itemscope itemtype="http://schema.org/HowToTool">Spare tire</div>
<div itemprop="tool" itemscope itemtype="http://schema.org/HowToTool">Lug wrench</div>
<div itemprop="tool" itemscope itemtype="http://schema.org/HowToTool">Jack</div>
<div itemprop="tool" itemscope itemtype="http://schema.org/HowToTool">Wheel wedges</div>
<div itemprop="supply" itemscope itemtype="http://schema.org/HowToSupply">Flares</div>
```

**Comment by AymenLoukil** [comment]
Yes it should be HowToTool of simply text value (HowToSupply too)

**Comment by RichardWallis** [comment]
Released in V3.4
